hermes/src/config.rs

```diff
@@ -49,6 +49,10 @@ pub struct RunOptions {
     /// Metrics Options
     #[command(flatten)]
     pub metrics: metrics::Options,
+
+    /// Update interval for price_feeds_cache in seconds. Default is 600 seconds.
+    #[arg(long = "price-feeds-cache-update-interval", default_value_t = 600)]
+    pub price_feeds_cache_update_interval: u64,
```

Comment:
I think we don't need to make this configurable but if we do, Pythnet module is a better place.